components/raftstore/src/store/worker/read.rs

```diff
@@ -566,6 +632,46 @@ impl Progress {
     }
 }
 
+struct SnapCache<E>
+where
+    E: KvEngine,
+{
+    cached_read_id: Option<ThreadReadId>,
+    snapshot: Box<Option<Arc<E::Snapshot>>>,
```

Comment:
Why put it in `Box`?

Reply:
It is insider Box before. I think I can remove it.